src/colorizer/utils/json_utils.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Replaces all NaN in string text (such as the string representation of a JSON
+ * object) with null. Can be used to safely parse JSON objects with NaN values.
+ */
+export const nanToNull = (json: string): string => json.replace(/NaN/g, "null");
```

Comment:
Had to be moved to its own file because of a dependency loop:
`urlLoadWorker` -> `data_utils` -> `Dataset` -> `UrlArrayLoader` -> `urlLoadWorker`